.github/workflows/codesee-diagram.yml

```diff
@@ -1,9 +1,7 @@
 on:
-  push:
-    paths-ignore:
-      - 'docs/**'
-    branches:
-      - main
+  workflow_dispatch:
+  schedule:
+    - cron: '25 0/12 * * *' # Build once every 12 hours, starting at 25 minutes past the hour (UTC)
```

Comment:
We do not target the hour because that is typically a high-demand slot; instead, we offset the run by 25 mins.